locales/fr.json

```diff
@@ -160,36 +160,36 @@
   "prompt.keys_removed": "{0} clés retirés",
   "prompt.keystyle_flat": "Modèle plat",
   "prompt.keystyle_flat_example": "par exemple : { \"a.b.c\": \"...\" }",
-  "prompt.keystyle_nested": "Modèle emboîté",
-  "prompt.keystyle_nested_example": "par exemple : { \"a\": { \"b\": { \"c\": \"...\" } } }",
+  "prompt.keystyle_nested": "Modèle emboité",
+  "prompt.keystyle_nested_example": "par exmple : { \"a\": { \"b\": { \"c\": «...» } } }",
   "prompt.keystyle_select": "Quel modèle souhaitez-vous utiliser pour organiser vos fichiers de la locale ?",
   "prompt.locales_dir_not_found": "Répertoire de la locale non trouvé. i18n Ally est désactivé.",
   "prompt.new_key_path": "Entrez le chemin de la nouvelle Clé.",
   "prompt.no_locale_loaded": "Aucun fichier locale chargé. Quelque chose s'est probablement mal passé avec la configuration de votre projet.",
   "prompt.replace_text_as": "Remplacer le texte par :",
   "prompt.select_display_locale": "Choisir la langue d'affichage. ({0})",
   "prompt.select_file_to_open": "Choisissez le fichier à ouvrir",
-  "prompt.select_file_to_store_key": "Sélectionnez le fichier pour stocker la Clé \"{0}\"",
+  "prompt.select_file_to_store_key": "Sélectionnez le fichier pour stocker la clé «{0}»",
   "prompt.select_source_language_for_translating": "Sélectionnez la langue source pour la traduction. ({0})",
   "prompt.select_source_locale": "Choisissez la langue source ({0})",
   "prompt.show_error_log": "Montrer les rapports",
   "prompt.star_on_github": "Mettre une étoile sur Github",
   "prompt.support": "i18n Ally est disponible gratuitement pour tous, si vous la trouver utile, considérez que vous nous supportez. Merci ! ❤",
   "prompt.translate_cancelled_multiple": "Traduction de {0} clés annulé.",
   "prompt.translate_done_multiple": "{0} Clé(s) traduite(s).",
-  "prompt.translate_done_single": "Traduction de \"{0}\" terminé.",
+  "prompt.translate_done_single": "Traduction de «{0}» terminé.",
   "prompt.translate_edit_translated": "Éditer le résultat",
   "prompt.translate_failed_multiple": "{0} clés n'ont pu être traduites.",
   "prompt.translate_failed_single": "{0} ne peut être traduire en {1}",
   "prompt.translate_in_progress": "Traduire",
   "prompt.translate_multiple_confirm": "Êtes-vous sûr de vouloir traduire {0} clés?",
   "prompt.translate_no_jobs": "Aucune traduction requise.",
-  "prompt.writing_js": "Le mode \"Écriture\" n'est pas disponible pour les fichiers JS/TS.",
-  "refactor.extract_ignore": "Ignorer la détection de \"{0}\" globalement",
-  "refactor.extract_ignore_by_file": "Ignorer la détection de \"{0}\" pour le fichier actuel",
+  "prompt.writing_js": "Le mode «Ecriture» n'est pas disponible pour les fichiers JS/TS.",
+  "refactor.extract_ignore": "Ignorer globalement la détection de «{0}».",
+  "refactor.extract_ignore_by_file": "Ignorer la détection de «{0}» pour le fichier en cours.",
```

Comment:
`Écriture` to `Ecriture`: is it on purpose?

Reply:
Écriture is good.